packages/framework/presence/src/cjs/package.json

```diff
@@ -6,6 +6,10 @@
 			"types": "./index.d.ts",
 			"default": "./index.js"
 		},
+		"./alpha": {
+			"types": "./alpha.d.ts",
+			"default": "./index.js"
+		},
 		"./internal": "./index.js",
```

Comment:
Remove "." and/or "./internal" to match removals in package.json.

Reply:
Hasn't been fixed - this should look very similar to root packages.json exports - all the same keys